### PR TITLE
Option picker for Masking Patient values

### PR DESCRIPTION
**Pull request by julianxcarter**
This PR introduces a picker for the `mask` constructorArg within the patientExtractor config section of the Config Editor. Though they're not currently functional, the mask values [that will soon be added to the MEF](https://github.com/mcode/mcode-extraction-framework/pull/155) are included options as well.

### CODE REVIEW

**Comment by Dtphelan1** [comment]
DP1 - We could probably update the MEF version now 😄

**Comment by Dtphelan1** [comment]
DP2 - Needs a rebase

**Comment by Dtphelan1** [comment]
DP3 - Small thing, but I noticed that if you click on the label of a checkbox (e.g. the text-label `telecom` next to the checkbox) it alternates the checked-value for the wrong field. After doing some digging, I learned that providing an `id` prop to the Form.Check component will solve this issue (see https://github.com/react-bootstrap/react-bootstrap/issues/4056#issuecomment-755458634). 

tl;dr let's add an `id` prop to the Form.Check component that's identical with the `key` prop.
